scss/popup.scss

```diff
@@ -1,9 +1,10 @@
 @use '../node_modules/draft-js/dist/Draft.css';
 @use '../node_modules/emoji-mart/css/emoji-mart.css';
-@use './components/images.scss';
+@import '../node_modules/spinthatshit/src/loaders.scss';
+@import './colors';
+@import './components/images.scss';
```

Comment:
Why not using `use`?

Reply:
use is for external css that you want to inline, import allows you to refer to scss variables